xla/service/xla_compile.bzl

```diff
@@ -71,6 +71,7 @@ def xla_aot_compile_gpu(
         # copybara:comment_begin(oss-only)
         target_compatible_with = select({
             "@local_config_cuda//:is_cuda_enabled": [],
+            "@local_config_rocm//:_enable_rocm": [],
```

Comment:
This doesn't seem to work with the CPU build. I am not sure why. Would it be ok if we merge this PR without the changes in xla_compile.bzl?

Reply:
Thank you for your message. Yes, it is ok to merge the PR without the changes in xla_compile.bzl.